datapy/__main__.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python3
+"""
+Main entrypoint for datapy
+"""
+
+from datapy.Schema import Schema
+from datapy.Table import Table
+
+if __name__ == "__main__":
+    # Create a Schema
+    Schema('Julia').create_schema()
+    # Create a Table instance
```

Comment:
some comment about this being test usage and what the data directory is for (even if that's just for current dev usage) would be helpful.